manifest.yaml

```diff
@@ -105,6 +105,21 @@ box:
       linux-arm64: victoria-metrics-linux-arm64-v1.79.1.tar.gz
     outputPath: lp-victoria-metrics
     skipManifestUpdate: true
+  - name: ffmpeg
+    strategy:
+      download: github
+      project: eugeneware/ffmpeg-static
+      commit: 7407c144d1e7b3ad93c6ef029827051c23c5e094
+    release: b6.0
+    skipGpg: true
+    skipChecksum: true
+    srcFilenames:
+      linux-amd64: ffmpeg-linux-x64
+      linux-arm64: ffmpeg-linux-arm64
+      darwin-amd64: ffmpeg-darwin-x64
+      darwin-arm64: ffmpeg-darwin-arm64
+    outputPath: ffmpeg
+    skipManifestUpdate: true
```

Comment:
Please could you add ffprobe in too so they're on the same version?

Reply:
Then I guess we can remove installing ffmpeg from apt?

Reply:
creating a cyclic loop: https://github.com/iameli-streams/catalyst/pull/2 :D

Reply:
Just merged, that PR, will merge this one when CI passes

Reply:
I'm wondering whether the CI test failures are because of removing the ffmpeg apt install 😕 
Tried to prove that with https://github.com/iameli-streams/catalyst/pull/3
@iameli @hjpotter92